packages/frontend/amp/components/BodyLiveblog.tsx

```diff
@@ -31,6 +32,39 @@ const body = (pillar: Pillar, tone: StyledTone) => css`
     }
 `;
 
+const updateButtonStyle = css`
```

Comment:
Definitely a candidate for a standardised button style in GUUI, not sure if we got anywhere with that yet though.